src/App.js

```diff
@@ -1,23 +1,10 @@
-import logo from './logo.svg';
 import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Welcome to Learning React.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+      <Calculator />
     </div>
   );
 }
```

Comment:
- [ ] Please, It would be better to use class-based components as the requirements said.

![class-based](https://user-images.githubusercontent.com/79876959/144699179-dc2d1d83-de2e-4f65-bc5a-ee2b231441d0.PNG)

Reply:
Fixed this problem.